containers/Dockerfile.base-spack

```diff
@@ -78,7 +81,15 @@ SHELL ["/bin/bash", "-c"]
 # Install spack
 RUN git clone -c feature.manyFiles=true https://github.com/spack/spack.git ${SPACK_ROOT} --branch ${SPACK_REPO_VERSION} --single-branch --depth=1
 
-RUN sed -i 's/granularity: microarchitectures/granularity: generic/' ${SPACK_ROOT}/etc/spack/defaults/concretizer.yaml
+# Install ACCESS-NRI's spack_config repo
+RUN git clone  https://github.com/ACCESS-NRI/spack_config.git ${SPACK_CONFIG_REPO_ROOT} --branch ${SPACK_CONFIG_REPO_VERSION}
+
+RUN ln -s ${SPACK_CONFIG_DIR}/concretizer.yaml ${SPACK_ROOT}/etc/spack/ \
+ && ln -s ${SPACK_CONFIG_DIR}/config.yaml ${SPACK_ROOT}/etc/spack/ \
+ && ln -s ${SPACK_CONFIG_DIR}/repos.yaml ${SPACK_ROOT}/etc/spack/
+# We don't want to use, only relevant for Gadi:
+#     ${SPACK_CONFIG_DIR}/packages.yaml
+#     ${SPACK_CONFIG}/linux/compilers.yaml
```

Comment:
If we don't want to use this commented code as part of the CI pipeline (which doesn't deal with Gadi), should it be here?
On Gadi, how would the un-commented code work?

Reply:
It's not code, it's a documentation.

Reply:
I see, resolved.